spec/controllers/api/sequences/sequences_create_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+describe Api::SequencesController do
+
+  include Devise::TestHelpers
+
+  describe '#create' do
+
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'creates a new sequences for a user' do
+      sign_in user
+      inputs = {name: "Scare Birds",
+                steps: [{
+                    message_type: 'move_rel'
+                  }]}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Space inside } missing.